audit/auditlog_test.go

```diff
@@ -50,7 +52,7 @@ func createVulnerabilities(num int) (vulnerabilities []types.Vulnerability) {
 }
 
 func createVulnerability() (vulnerability types.Vulnerability) {
-	vulnerability.Cve = "7.8"
+	vulnerability.Cve = "nerabil"
```

Comment:
@DarthHater FWIW, my use of the string "nerabil" in unit tests was intended for matching/filtering against the `Vulnerability.Title` structure element, which contained the string "Vulnerability". That may have been a mistake if the intent was to filter against Vulnerability.Cve element instead. I'm a little confused about that "nerabil" sub-string being used as the Cve element value.

Reply:
Changed the values in 526f570